### PR TITLE
Update Tomcat versions to the latest #560

### PR DESCRIPTION
**Pull request by ahmadshalabi**
I update tomcat versions to the latest. Also, I set the default to 10.0.14.

### CODE REVIEW

**Comment by ahmadshalabi** [comment]
`Error: Exception in thread "main" com.github.mongobee.exception.MongobeeException: Invalid url: https://repository.apache.org/content/repositories/snapshots/org/apache/maven/apache-maven/4.0.0-alpha-1-SNAPSHOT/apache-maven-4.0.0-alpha-1-20211230.155705-237-bin.zip`

The issue is the current maven-4.0.0-alpha-1 no longer exists in the apache repository and this will cause a failure when someone wants to try the alpha version.


CC: @eddumelendez @marc0der

File: src/main/scala/io/sdkman/changelogs/TomcatMigration.scala
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/TomcatMigration.scala:41]: Why not keep the legacy versions? SDKMAN is specifically for allowing parallel version of any SDK.

Comment (ahmadshalabi) [reply src/main/scala/io/sdkman/changelogs/TomcatMigration.scala:41]: Thanks for reminding me to keep the legacy versions.

**Comment by marc0der** [comment]
If the maven team don't understand that resources on the web should be immutable, I don't see the point of hosting these 4 alpha binaries at all. We'll continually be playing a wack-a-mole game trying to keep them up to date. wdyt @helpermethod?

**Comment by helpermethod** [comment]
@marc0der It was probably a bad idea from my side to add them. I guess we should just remove them and wait for a stable release.

**Comment by marc0der** [comment]
@helpermethod I thought your idea was great, I just wasn't expecting how they would tamper with web resources. Especially since they are maven 😄

**Comment by marc0der** [comment]
@helpermethod I thought your idea was great, I just wasn't expecting how they would tamper with web resources. Especially since they are maven 😄

File: src/main/scala/io/sdkman/changelogs/TomcatMigration.scala
Comment (ahmadshalabi) [reply src/main/scala/io/sdkman/changelogs/TomcatMigration.scala]: I did the change and added the last 10.1.0 version (10.1.0-M8).

**Review by marc0der — APPROVED** [review]

**Comment by marc0der** [comment]
We might need to remove the maven 4 migration to get the build green before merging.

**Comment by ahmadshalabi** [comment]
@marc0der, I opened #1033 to track the deletion of the maven 4 version.

**Comment by marc0der** [comment]
Thanks for your contribution @ahmadshalabi

**Comment by ahmadshalabi** [comment]
> Thanks for your contribution @ahmadshalabi

Most welcome @marc0der.

**Pull request merged**
